### DISCUSSION TITLE
Vercel Postgres Compute Time Running Nonstop for August 31, 2023 17:00 until September 01, 2023 08:31 (and Bot Visits)

### DISCUSSION THREAD

**Discussion by josephna76** [vercel/community#3886] [Help] [answered]
### Summary

I have a Vercel Project in development that clocked in 15.21 computer hours in the last day, which only had 15.5 total hours pass. 
I took the last two days off and had nearly 0 traffic, and the usage is exponentially higher than my average usage of around 0.8 hours/day when actively developing. Is this a bug in the dashboard/service or is something else going on?

As a side note a few days ago I got a surge of bot traffic despite not actively advertising the url of the webpage. Has anyone else noticed this? I know that the bots did not cause the computer hours surge because they all bounced off the login page and did not access pages that interact with the db.

<img width="950" alt="Screenshot 2023-09-01 at 8 41 54 AM" src="https://github.com/vercel/community/assets/59426134/5c026bbd-0888-4f81-8144-42f19e9323e6">
_Compute Hours_
<img width="950" alt="Screenshot 2023-09-01 at 8 42 04 AM" src="https://github.com/vercel/community/assets/59426134/e6e56244-fe76-4113-9159-27ba5559907e">
_Data Transfer_
<img width="1241" alt="Screenshot 2023-09-01 at 8 45 35 AM" src="https://github.com/vercel/community/assets/59426134/3839a866-bebd-4049-b726-96dc0928dc2b">
_(side note regarding the linux bots)_

### Steps to Reproduce

I only have about 5 columns of very small amounts of test data in about 4 tables, these are used for testing.
I have not made any significant changes to my db or invoked heavy usage (see visitor chart to see the approximate amount of non-usage the last few days, compared to the exceptionally high compute hours used).

**Comment by kevin12244** [comment]
My usage also spiked as well.
![usage spike](https://github.com/vercel/community/assets/55848436/62ed92d1-5afa-4d69-b90a-b56facbbbe64)

**Reply by josephna76** [reply]
That's an immense amount of compute time. Was this still within your total compute time allotment or did this send you well over the limit? I've pasted the ratios of my monthly usage, but my 15 hours took up about 25% of my total monthly allotment.
<img width="918" alt="Screenshot 2023-09-01 at 9 16 04 AM" src="https://github.com/vercel/community/assets/59426134/e00dfc47-d6ca-4afa-882b-bd195f0290bc">

**Reply by kevin12244** [reply]
It got me well over the limit. My limit was 100 hrs. Most of the compute time prior was probably from me accessing the database myself. My website has some visitors (usually not more than 30 per days), but it is still in production so most of the usage is probably from me testing stuff out. Additionally, there was also around 30 visitors today.

**Comment by amyegan** [comment]
Thanks for reporting this. The team is investigating now 🙇

**Reply by amyegan** [reply, marked as answer]
Some users are seeing higher than expected usage of Vercel Postgres, and we believe that this is in error. While we continue to investigate, we will update the dashboard to remove these erroneous metrics. We apologize for the inconvenience.

**Reply by josephna76** [reply]
Thank you Amy, it's been resolved on my end.
